### PR TITLE
fix: `loaderUtils.parseQuery()` deprecation warning (`loader-utils`)

### PR DESCRIPTION
**Pull request by mitkaaa**
Special fix for the latest version of the loader-utils(^1.1.0)
Removes warning when transpiling the webpack

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sudo: false
```
Comment (evilebottnawi) [inline .travis.yml]: Don't add `.travis.yml` in this PR, we use `webpack-defaults` for all repositories https://github.com/webpack-contrib/webpack-defaults

Comment (mitkaaa) [inline .travis.yml]: ok

File: index.js [hunk 2]
```diff
@@ -59,10 +59,7 @@ function getExtractedSVG(svgStr, query) {
 function SVGInlineLoader(content) {
     this.cacheable && this.cacheable();
     this.value = content;
-    // Configuration
-    var query = loaderUtils.parseQuery(this.query);
-
```
Comment (d3viant0ne) [inline index.js]: This should still be required. `var query = loaderUtils.getOptions(this) || {};`

Comment (michael-ciniawsky) [inline index.js]: `const options = loaderUtils.getOptions(this)` && global rename `query` => `options`

**Comment by mitkaaa** [comment]
I checked it works :)

**Comment by d3viant0ne** [comment]
This is going to need to be rebased

**Comment by le0m** [comment]
is there any update on this?

**Comment by d3viant0ne** [comment]
@le0m - I'm going to take care of this in a separate PR as the loader utils change is not backwards compatible. 

With Webpack 4 on the way, we are a bit buried at the moment, i'll try and get a beta build that handles the deprecation warning & Webpack 4 support before the end of the week.

**Comment by jhnns** [comment]
Fixed by https://github.com/webpack-contrib/svg-inline-loader/pull/72/files

**Comment by vidal7** [comment]
Yeah! Thank you. @jhnns, I supposed that there will be a release soon?

**Comment by vidal7** [comment]
@jhnns, I still don't see the release on npmjs

**Pull request closed without merging**